### ISSUE TITLE
bug: request failed to execute 'clone' on 'Request': Request body is already used

### ISSUE THREAD

**Issue by lazyvab** [requestly/requestly#688]
### Describe your issue?

`Modify Request body` rule fails to execute when the code looks like:

```js
function modifyRequestBody(args) {
  const { method, url, body, bodyAsJson } = args;
  bodyAsJson.newProperty = false;
  return JSON.stringify(bodyAsJson);
}
```
Error:
`request failed to execute 'clone' on 'Request': Request body is already used`

### Repro steps

See issue description.

### What Requestly tool were you using?

- [X] Extension
- [ ] Desktop-App
- [ ] Android SDK
- [ ] Selenium

### Your Environment

Chrome

### Requestly Version

23.6.5

**Comment by lazyvab** [comment]
This seems like a regression introduced with the recent extension release `23.6.5`. Looking into this on priority. 

Meanwhile, anyone who is blocked can revert to previous extension version this way:

1. Download previous extension build [v23.5.19](https://github.com/requestly/requestly/blob/master/browser-extension/mv2/builds/chrome/Requestly_23.5.19_2023-05-19.zip).
2. Go to chrome://extensions/.
3. Enable `Developer mode`.
4. Drag the downloaded zip to the page.
5. Disable official extension version `23.6.5`.

Once this is fixed, remove the extension version `23.5.19` and enable back the official extension.

**Comment by lazyvab** [comment]
The bug has been fixed. A new version of the extension will be released. `23.6.7`.

Here is the preview build: https://github.com/requestly/requestly/blob/master/browser-extension/mv2/builds/chrome/Requestly_23.6.7_2023-06-07.zip

In my previous comment, instead of installing older version, this newer version can be used.

I'll update the thread when it gets published in Chrome web store.
